iTunesUDataPurge/RemoveITunesUCourses.java

```diff
@@ -592,18 +611,18 @@ public static SOAPMessage createLoginRequest(String serverURI, String username,
 		return soapMessage;
 	}
 
-	public static String getSessionId(SOAPMessage soapResponse ) {
-		String sessionId = "sessionId";
+	public static String getFirstNode(SOAPMessage soapResponse ) {
```

Comment:
It returns the first node's text content. Can the function be renamed as getFirstNodeText?